lib/shared/ajv.js

```diff
@@ -8,27 +8,21 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-const Ajv = require("ajv"),
-    metaSchema = require("ajv/lib/refs/json-schema-draft-04.json");
+const Ajv = require("ajv-draft-04");
 
 //------------------------------------------------------------------------------
 // Public Interface
 //------------------------------------------------------------------------------
 
 module.exports = (additionalOptions = {}) => {
     const ajv = new Ajv({
-        meta: false,
         useDefaults: true,
         validateSchema: false,
-        missingRefs: "ignore",
         verbose: true,
-        schemaId: "auto",
+        strict: "log",
+        strictTuples: false,
```

Comment:
```suggestion
        strict: false,
```

If we set `strict: false`, then we don't need to explicitly set `strictTuples: false`?

Reply:
Correct